### ISSUE TITLE
Forest hiccups while following chain

### ISSUE THREAD

**Issue by elmattic** [ChainSafe/forest#2278]
**Describe the bug**
While running forest node, I'm facing some issues that seem to block forest following mainnet chain.

Forest was stucked at 2,370,516:
 
```
$ ./target/release/forest-cli sync status
sync status:
Base:   [bafy2bzaceco5hg3557wnsd53eqcqkrsxuatyu2ohtm26avlbcrbayjlndap6a, bafy2bzacec3vma2gnru4q6jym3ndu4dcgigbqmpmyjmp2swv52vmspuz4wsus]
Target: [bafy2bzacedmkek6c4qtx3ceowqptzhep3kkdbg6vtfle6kyew3pmd4mpesrhe, bafy2bzaceavdmjvlry6oujcwbujvonlkwllk2tp7sbwrthchdhk33ckoriqug, bafy2bzacechsttci4zsvcvkjayvug35d2ae7jxhtjtrqxprlhh4hg77w36vuq, bafy2bzacebxkrdslcxy2ghtcdeaqdguhxs2ar3ppnsldqxugvncrxvcebnb62, bafy2bzacecskr7zuhbeaw25av5hj7hbrkso5ibmhqjjk6dhjhp6w3zembikjy, bafy2bzaceamollqacug4hfelqonv3aydnppfaxcbqul6hp55pj6id4m2hfuua, bafy2bzacebzonge32sncfos7r6aunjv2iuzjvmcd5xweluing3f5riih6gw4y] (2370523)
Height diff:    7
Stage:  message sync
Height: 2370516
Elapsed time:   32641s
```

Restarting the node seems to help catching up again, but by experience the issue can arise later on.

**To Reproduce**

1. Init db with a recent snapshot
2. Run Forest for a little while
3. Wait till it stop syncing

**Log output**

[other_forest.log](https://github.com/ChainSafe/forest/files/10105025/other_forest.log)

**Expected behaviour**

Forest can follow the network.

**Environment (please complete the following information):**
 - OS: macOS
 - Rust version: 1.66.0-nightly
 - Branch: b7b8202d8c0eec9bf4976dc089dafc78c2c6ea61+paritydb

**Other information and links**

**Comment by elmattic** [comment]
It is happening again.

```
 2022-11-25T17:01:10.016Z WARN  forest_chain_sync::tipset_syncer > Validating block [CID = bafy2bzaceajuhsl5u27lxr4e5lgjwdjs2gynltgbfxxyvxpepgcv22bucoqsk] in EPOCH = 2369642 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceaxs5cojltcr42b744m5afnrtmo4bjilqlppz445hhjn3sit76yjs, StateManager error: failed to find state tree bafy2bzaceb7ntgyhin5pesiwymdzwmcp7rk6bmc3prymhrbfwq5a33j542fs6, Processing error: Error calculating weight: failed to find state tree bafy2bzaceb7ntgyhin5pesiwymdzwmcp7rk6bmc3prymhrbfwq5a33j542fs6, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceaxs5cojltcr42b744m5afnrtmo4bjilqlppz445hhjn3sit76yjs, Validation error: Consensus error: StateManager error: failed to find state tree bafy2bzaceb7ntgyhin5pesiwymdzwmcp7rk6bmc3prymhrbfwq5a33j542fs6, Consensus error: StateManager error: failed to find state tree bafy2bzaceb7ntgyhin5pesiwymdzwmcp7rk6bmc3prymhrbfwq5a33j542fs6
 2022-11-25T17:01:10.016Z ERROR forest_chain_sync::tipset_syncer > Sync messages check state failed for single tipset
 2022-11-25T17:01:10.016Z ERROR forest_chain_sync::tipset_syncer > Syncing tipset range [2369640, 2369642] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceaxs5cojltcr42b744m5afnrtmo4bjilqlppz445hhjn3sit76yjs, StateManager error: failed to find state tree bafy2bzaceb7ntgyhin5pesiwymdzwmcp7rk6bmc3prymhrbfwq5a33j542fs6, Processing error: Error calculating weight: failed to find state tree bafy2bzaceb7ntgyhin5pesiwymdzwmcp7rk6bmc3prymhrbfwq5a33j542fs6, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceaxs5cojltcr42b744m5afnrtmo4bjilqlppz445hhjn3sit76yjs, Validation error: Consensus error: StateManager error: failed to find state tree bafy2bzaceb7ntgyhin5pesiwymdzwmcp7rk6bmc3prymhrbfwq5a33j542fs6, Consensus error: StateManager error: failed to find state tree bafy2bzaceb7ntgyhin5pesiwymdzwmcp7rk6bmc3prymhrbfwq5a33j542fs6
```

**Comment by elmattic** [comment]
Just reproduced it now:

```
 2022-12-02T15:53:55.613Z INFO  forest_chain_sync::tipset_syncer > Validating tipset: EPOCH = 2389665, N blocks = 5
 2022-12-02T15:53:56.508Z INFO  forest_chain_sync::tipset_syncer > Validating tipset: EPOCH = 2389667, N blocks = 8
 2022-12-02T15:53:56.563Z WARN  forest_chain_sync::tipset_syncer > Validating block [CID = bafy2bzacedamig4kjk2sj6djdtv6ix46druwlgfhuvqxdhzdninpsijyx7jfu] in EPOCH = 2389667 failed: Validation error: StateManager error: failed to find state tree bafy2bzacedv4s7l5niyuukci5tchletf6tcw7qeyx4ddg6qxyz3e6u4lhnggs, Processing error: Error calculating weight: failed to find state tree bafy2bzacedv4s7l5niyuukci5tchletf6tcw7qeyx4ddg6qxyz3e6u4lhnggs, Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceaokz44ale2q6pvns2ghnuslj4fqjs3ipe5oyiehbhilankadcylk, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceaokz44ale2q6pvns2ghnuslj4fqjs3ipe5oyiehbhilankadcylk, Validation error: Consensus error: StateManager error: failed to find state tree bafy2bzacedv4s7l5niyuukci5tchletf6tcw7qeyx4ddg6qxyz3e6u4lhnggs, Consensus error: StateManager error: failed to find state tree bafy2bzacedv4s7l5niyuukci5tchletf6tcw7qeyx4ddg6qxyz3e6u4lhnggs
 2022-12-02T15:53:56.563Z ERROR forest_chain_sync::tipset_syncer > Sync messages check state failed for single tipset
 2022-12-02T15:53:56.564Z ERROR forest_chain_sync::tipset_syncer > Syncing tipset range [2389665, 2389667] failed: Validation error: StateManager error: failed to find state tree bafy2bzacedv4s7l5niyuukci5tchletf6tcw7qeyx4ddg6qxyz3e6u4lhnggs, Processing error: Error calculating weight: failed to find state tree bafy2bzacedv4s7l5niyuukci5tchletf6tcw7qeyx4ddg6qxyz3e6u4lhnggs, Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceaokz44ale2q6pvns2ghnuslj4fqjs3ipe5oyiehbhilankadcylk, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceaokz44ale2q6pvns2ghnuslj4fqjs3ipe5oyiehbhilankadcylk, Validation error: Consensus error: StateManager error: failed to find state tree bafy2bzacedv4s7l5niyuukci5tchletf6tcw7qeyx4ddg6qxyz3e6u4lhnggs, Consensus error: StateManager error: failed to find state tree bafy2bzacedv4s7l5niyuukci5tchletf6tcw7qeyx4ddg6qxyz3e6u4lhnggs
```

**Comment by elmattic** [comment]
I've managed to reproduce this issue on Fedora Linux 37 as well.

**Comment by LesnyRumcajs** [comment]
We have bunch of those on rocksdb as well, filtered log from the sync check.
```
[chainsafe@forest-test ~]$ docker logs forest-mainnet | grep " Invalid tipset: no msg root with ci"
2023-02-21T10:06:04.952164Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzaceattme2twkhnx4sdxxqwxtw2sb72vrr6v2dnhbvgjfiqlvwjttoo4] in EPOCH = 2622252 failed: Validation error: Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:06:04.952226Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622250, 2622252] failed: Validation error: Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:06:05.614595Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzacedefp6ivjsruh6qvbuhrhfuywqn46cr77qau6gsipjuflmufiyha6] in EPOCH = 2622252 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:06:05.614647Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622250, 2622252] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:06:06.670375Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzaceacecxuaftyqalhxhul4oa4wj5djbc57wqoqpv5ju452nqvawymc6] in EPOCH = 2622252 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:06:06.670440Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622250, 2622252] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebsapayvtebapkxaefinuzvr6vnbarpe26aaetitwo2pz6hky4rw6, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:16:35.062209Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzacecuk6noi2xfoeqveucmiyjqeyqdppx4nwfkdadudtem4k7db7j5lo] in EPOCH = 2622273 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:16:35.062266Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622271, 2622273] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:16:36.094930Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzacecme6bafkz4t7ijmxqjqqwa7umy6juszt4pvckttkjloy2emkaxvg] in EPOCH = 2622273 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:16:36.094973Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622271, 2622273] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:16:36.984943Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzacebttk3w2hdinafwxr4jw7u6rm23q3pnma7zznouobrbitchcqheg4] in EPOCH = 2622273 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:16:36.985002Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622271, 2622273] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceboclsgn2lvvpb7j6oeqzcardnn5lubbuqcfwg5en364zljgdhk7q, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:23:04.666533Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzaceb7fupyiyzk2eobxymudjt64oqofd4rgzjp54ieyf4stxemt7ygzm] in EPOCH = 2622286 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceajzodnoszffml4vrjhesbqgckd3jsajgj5yhs7wh6lx25ven3bdu, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceajzodnoszffml4vrjhesbqgckd3jsajgj5yhs7wh6lx25ven3bdu, StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:23:04.666587Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622284, 2622286] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceajzodnoszffml4vrjhesbqgckd3jsajgj5yhs7wh6lx25ven3bdu, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceajzodnoszffml4vrjhesbqgckd3jsajgj5yhs7wh6lx25ven3bdu, StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:26:06.656537Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzacedi5j2z2ogcm74fuwgwjr2iitarmgeli53x2fhgvcmd46zftzqf76] in EPOCH = 2622292 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebqxnbx4tloioyrsa47b4bta27e24hfclebqb5mrgne5oeg7zi4nu, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebqxnbx4tloioyrsa47b4bta27e24hfclebqb5mrgne5oeg7zi4nu, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:26:06.656589Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622290, 2622292] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebqxnbx4tloioyrsa47b4bta27e24hfclebqb5mrgne5oeg7zi4nu, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebqxnbx4tloioyrsa47b4bta27e24hfclebqb5mrgne5oeg7zi4nu, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:26:18.164334Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzacedi5j2z2ogcm74fuwgwjr2iitarmgeli53x2fhgvcmd46zftzqf76] in EPOCH = 2622292 failed: Validation error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebqxnbx4tloioyrsa47b4bta27e24hfclebqb5mrgne5oeg7zi4nu, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebqxnbx4tloioyrsa47b4bta27e24hfclebqb5mrgne5oeg7zi4nu, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:26:18.164397Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622290, 2622292] failed: Validation error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzacebqxnbx4tloioyrsa47b4bta27e24hfclebqb5mrgne5oeg7zi4nu, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzacebqxnbx4tloioyrsa47b4bta27e24hfclebqb5mrgne5oeg7zi4nu, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:27:05.649341Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzacea7z466pwlolyc2mixscyzmq22kzbgtdqmecblx22bxxsyc37nqb6] in EPOCH = 2622294 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:27:05.649389Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622292, 2622294] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version., Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:27:06.106711Z  WARN forest_chain_sync::tipset_syncer: Validating block [CID = bafy2bzacebck7xwszhk3qxe4sc4qcxemfxuxot5wyoutxg22l63fhahahwenq] in EPOCH = 2622294 failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Processing error: Could not update state: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
2023-02-21T10:27:06.106760Z ERROR forest_chain_sync::tipset_syncer: Syncing tipset range [2622292, 2622294] failed: Validation error: Validation error: Could not compute base fee: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Processing error: Error calculating weight: Can't create a valid state tree from the given root. This error may indicate unsupported version., Processing error: Failed to calculate state: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Processing error: Could not update state: Invalid tipset: no msg root with cid bafy2bzaceack5fkg2k26ng2ya2du2x3mb5e7rzhw7yreksgiw2vj45s5uu2hm, Validation error: Consensus error: Power actor not found, Consensus error: StateManager error: Can't create a valid state tree from the given root. This error may indicate unsupported version.
```
